test/timerfd/main.c

```diff
@@ -0,0 +1,69 @@
+#include <stdio.h>
+#include <sys/timerfd.h>
+#include <sys/select.h>
+#include <time.h>
+#include "test.h"
+
+// ============================================================================
+// Test cases for timerfd full process
+// ============================================================================
+int test_timerfd() {
+    int tfd = timerfd_create(CLOCK_REALTIME,  0);
+
+    printf("Starting at (%d)...\n", (int)time(NULL));
+    if (tfd == 0) {
```

Comment:
if tfd < 0 , we should also throw_error, so the check shoud be 
```
if (tfd <= 0) {
THROW_ERROR("timerfd_create(CLOCK_REALTIME, ...) failed");
}
```

Reply:
Done